polyfill.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+module.exports = function getPolyfill() {
+	return Array.prototype.find || require('./implementation');
```

Comment:
There is a bug in early implementations of `Array#find` that this should be detecting - https://github.com/paulmillr/es6-shim/blob/master/es6-shim.js#L1432-L1440